### PR TITLE
feat(bmd): Pollworker system diagnostics screen (computer and printer only)

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Task: #1417
Depends on: https://github.com/votingworks/kiosk-browser/pull/98

Adds a system diagnostics screen to the BMD pollworker view. The screen is accessed by clicking the "System Diagnostics" button on the pollworker actions screen. To start, it has only two sections:
- Computer: battery and power cord status, which updates automatically every 3 seconds
- Printer: printer status and toner levels, which update when the screen loads or when the user clicks a button to refresh

Future PRs will add sections for the card reader and accessible controller, and also allow this screen to be accessed by the admin user.

## Demo Video or Screenshot

Screen Shot 2022-03-22 at 2 20 14 PM

Screen Shot 2022-03-22 at 4 57 07 PM

Screen Shot 2022-03-22 at 4 58 10 PM

Screen Shot 2022-03-22 at 4 59 59 PM

## Testing Plan 
TODO: add full automated tests for this screen
Manually tested extensively with a printer

## Checklist
- ~~[ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.~~ Will revisit logging after finishing all of the features of the diagnostic screen
- [x] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Comment by beausmith** [comment]
Looks great. Comments shared via video chat:
- Let's separate each bit of data onto it's own line and prefix with the [√] or (!) icon so that they page can be scanned to see aligned icons for success/warning.
- When there is a warning: Add text to instruct to the viewer if there is something they can do.

**Comment by jonahkagan** [comment]
Thanks @beausmith! I updated the screenshots with those changes

File: frontends/bmd/src/pages/diagnostics_screen.tsx [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  const [printerStatus, setPrinterStatus] = useState<PrinterStatusState>({
+    isLoading: true,
+  });
+
+  // devices.printer only updates when the printer connects/disconnects. We want
+  // to load its current status, since it may have changed (e.g. if it ran out
+  // of paper).
+  const loadPrinterStatus = useCallback(async () => {
+    setPrinterStatus({ isLoading: true });
+    const printer = await hardware.readPrinterStatus();
```
[the remark this answers is not in the document]
Comment (eventualbuddha) [reply frontends/bmd/src/pages/diagnostics_screen.tsx:134]: I'm not sure what actual problem it causes in prod, but the test warnings certainly are annoying. I'll try it myself and see why it's still complaining.

File: libs/test-utils/src/fake_kiosk.ts [hunk 2]
```diff
@@ -33,7 +47,9 @@ export function fakePrinterInfo(
     description: props.name ?? 'Fake Printer',
     isDefault: false,
     name: 'Fake Printer',
-    status: 3, // idle
+    state: 'idle' as KioskBrowser.IppPrinterState,
```
[the remark this answers is not in the document]
Comment (eventualbuddha) [reply libs/test-utils/src/fake_kiosk.ts:50]: https://github.com/votingworks/kiosk-browser/issues/108

[the remark this answers is not in the document]
Comment (jonahkagan) [reply libs/test-utils/src/fake_kiosk.ts]: Ok I fixed this on the vxsuite side by changing its type definition file for kiosk-browser

**Comment by jonahkagan** [comment]
@eventualbuddha I think I've addressed all your feedback

**Review by eventualbuddha — APPROVED** [review]

**Pull request merged**